k8s-tf-cluster/00-inputs.tf

```diff
@@ -0,0 +1,49 @@
+/*
+This file defines:
+- Required Terraform version
+- Required provider versions
+- Storage backend details
+- Input variables
+*/
+
+terraform {
+  required_version = ">= 0.12.3"
+
+  backend "remote" {
+    hostname     = "app.terraform.io"
+    organization = "k8s-infra"
+
+    workspaces {
+      // should match the name of the gcp project
+      name = "k8s-infra-dev-cluster-turnup"
+    }
+  }
+
+  required_providers {
+    google      = "~> 2.10"
+    google-beta = "~> 2.10"
+  }
+}
+
+variable "cluster_name" {
+  type        = string
+  description = <<EOT
```

Comment:
Just curious, why are you using here docs for the descriptions?

Reply:
Allows for multi-line descriptions. Only one variable uses it, but it keeps things consistent.